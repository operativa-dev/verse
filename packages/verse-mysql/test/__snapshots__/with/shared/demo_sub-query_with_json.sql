-- Executing SQL: Parameters: [$1='B%']
select `t1`.`ArtistId`, `t1`.`Name`, (
   select json_unquote(json_extract(json_extract(`t6`.`c0`, '$[0]'), '$[1]')) as `c1`
   from (
      select json_arrayagg(json_array(`t4`.`AlbumId`, `t4`.`Title`, `t4`.`ArtistId`)) as `c0`
      from `Album` as `t4`
      where `t4`.`ArtistId` = `t1`.`ArtistId`
   ) as `t6`
   where json_unquote(json_extract(json_extract(`t6`.`c0`, '$[0]'), '$[1]')) like ?
) as `c2`
from `Artist` as `t1`
limit 10

