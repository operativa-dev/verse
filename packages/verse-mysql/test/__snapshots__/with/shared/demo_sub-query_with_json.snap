[
  [
    Artist {
      "albums": [],
      "artistId": 1,
      "name": "AC/DC",
    },
    undefined,
  ],
  [
    Artist {
      "albums": [],
      "artistId": 2,
      "name": "Accept",
    },
    "Balls to the Wall",
  ],
  [
    Artist {
      "albums": [],
      "artistId": 3,
      "name": "Aerosmith",
    },
    "Big Ones",
  ],
  [
    Artist {
      "albums": [],
      "artistId": 4,
      "name": "Alanis Morissette",
    },
    undefined,
  ],
  [
    Artist {
      "albums": [],
      "artistId": 5,
      "name": "Alice In Chains",
    },
    undefined,
  ],
  [
    Artist {
      "albums": [],
      "artistId": 6,
      "name": "Ant√¥nio Carlos Jobim",
    },
    undefined,
  ],
  [
    Artist {
      "albums": [],
      "artistId": 7,
      "name": "Apocalyptica",
    },
    undefined,
  ],
  [
    Artist {
      "albums": [],
      "artistId": 8,
      "name": "Audioslave",
    },
    undefined,
  ],
  [
    Artist {
      "albums": [],
      "artistId": 9,
      "name": "BackBeat",
    },
    "BackBeat Soundtrack",
  ],
  [
    Artist {
      "albums": [],
      "artistId": 10,
      "name": "Billy Cobham",
    },
    undefined,
  ],
]